### PR TITLE
comment

### PR DESCRIPTION
**Pull request by oabdolah**

### CODE REVIEW

**Review by Saharnaz1 — COMMENTED** [review]
Looks great!

**Review by Saharnaz1 — APPROVED** [review]

File: src/test/java/com/project_name/step_definitions/Test.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.project_name.step_definitions;
+
+import com.project_name.utilities.Driver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
```
Comment (oabdolah) [inline src/test/java/com/project_name/step_definitions/Test.java:5]: i don't like this

Comment (Rilas2023) [reply src/test/java/com/project_name/step_definitions/Test.java:5]: what do you like?

**Review by alisonh7574 — COMMENTED** [review]
looks good

**Comment by alisonh7574** [comment]
looks good 2

**Review by alisonh7574 — APPROVED** [review]

**Pull request merged**